packages/dd-trace/src/appsec/iast/taint-tracking/plugin.js

```diff
@@ -139,12 +139,12 @@ class TaintTrackingPlugin extends SourceIastPlugin {
   addDatabaseSubscriptions () {
     this.addSub(
       { channelName: 'datadog:sequelize:query:finish', tag: SQL_ROW_VALUE },
-      ({ result }) => this._taintDatabaseResult(result, 'sequelize')
+      ({ result }) => this._taintDatabaseResult(result, 'sequelize', getIastContext(storage('legacy').getStore()))
```

Comment:
I think we should avoid binding the legacy store from AppSec. The long-term goal is that each product can have their own store and their own plugins with their own bindings, and altering the legacy store bindings in more than one place will make it much more difficult to refactor.

Reply:
Hey @rochdev this is for Sequelize which we are not changing in this PR, previously `_taintDatabaseResult` had `getIastContext(storage('legacy').getStore())` set in the parameters, I think this is just to keep sequelize the same since we are now passing the `currentStore` to pg in line 147 `({ result, currentStore }) => this._taintDatabaseResult(result, 'pg', getIastContext(currentStore))`